envs/conda_build_config.yaml

```diff
@@ -151,6 +159,8 @@ ninja:
   - 1.10.*
 nccl:
   - 2.8.*
+nlohmann_json:
```

Comment:
It looks like this project marks things deprecated in x.y, but doesn't actually remove them until new x releases. We may be safe with 3.* here.

Reply:
Same thought I'd for this as I mentioned for libdate. But I'm fine with changing it to 3.*.